sound/soc/sof/ipc4-topology.c

```diff
@@ -683,9 +685,6 @@ static int sof_ipc4_widget_setup_comp_pipeline(struct snd_sof_widget *swidget)
 		goto err;
 	}
 
-	/* TODO: Get priority from topology */
-	pipeline->priority = 0;
-
```

Comment:
If I take a look at the topologies then I see that there is not a single instance when the priority is not set to 0, is that OK for now?
How the issue should be fixed for AES?

Reply:
It really begs the question "how was this tested" if all existing priorities are zero, and how this can possibly fix the AEC issue.

I am not convinced at all @RanderWang, or you didn't include a pointer on topoogy changes....

Reply:
sorry I added the note in wrong PR. https://github.com/thesofproject/sof/pull/8204. Now AEC sets priority.

Reply:
How does this work if there are multiple pipelines with the same priority? Is this allowed? if not and this is an error, where is this trapped?